docs/DEVELOPER.md

```diff
@@ -84,6 +84,7 @@ You can find more info about debugging [tests with Bazel in the docs.](https://g
 
 - Run: `node tests/legacy-cli/run_e2e.js`
 - Run a subset of the tests: `node tests/legacy-cli/run_e2e.js tests/legacy-cli/e2e/tests/i18n/ivy-localize-*`
+- Run on a custom set of npm packages (tar files): `node tests/legacy-cli/run_e2e.js --package _angular_cli.tgz _angular_create.tgz dist/*.tgz ...`
```

Comment:
Would be helpful to write the equivalent of `node tests/legacy-cli/run_e2e.js tests/legacy-cli/e2e/tests/i18n/ivy-localize-*`.